Code/Framework/AzFramework/AzFramework/Quality/QualityCVarGroup.cpp

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzFramework/Quality/QualityCVarGroup.h>
+#include <AzFramework/Quality/QualitySystemBus.h>
+
+#include <AzCore/Settings/SettingsRegistryImpl.h>
+#include <AzCore/Settings/SettingsRegistryVisitorUtils.h>
+#include <AzCore/std/smart_ptr/unique_ptr.h>
+#include <AzCore/Console/IConsole.h>
+#include <AzCore/StringFunc/StringFunc.h>
+
+namespace AzFramework
+{
+    QualityCVarGroup::QualityCVarGroup(AZStd::string_view name, AZStd::string_view path)
+        : m_name(name)
+        , m_path(path)
+        , m_qualityLevel(QualityLevels::LevelFromDeviceRules)
+    {
+        using FixedValueString = AZ::SettingsRegistryInterface::FixedValueString;
+        FixedValueString description;
+
+        if (auto settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry != nullptr)
+        {
+            // optional default quality level
+            settingsRegistry->GetObject(m_qualityLevel, FixedValueString::format("%.*s/Default", AZ_STRING_ARG(path)));
+
+            // optional description
+            settingsRegistry->Get(description, FixedValueString::format("%.*s/Description", AZ_STRING_ARG(path)));
+        }
+
+        m_functor = AZStd::make_unique<QualityCVarGroupFunctor>(
+            m_name.c_str(),
+            description.c_str(),

```

Comment:
Definitely.
The description would need to be persisted in this class and declared before the `m_functor` so that its lifetime is at least as long as the ConsoleFunctorBase

Reply:
Good catch, I ran into this same problem with the name and didn't see description will have the same problem, though I'd prefer to be able to not have string copies and point to the string values in the SettingsRegistry when needed.